x/mint/abci_app.go

```diff
@@ -5,11 +5,17 @@ import (
 )
 
 // Inflate every block, update inflation parameters once per hour
-func BeginBlocker(ctx sdk.Context, k Keeper) {
+func BeginBlocker(ctx sdk.Context, k Keeper) error {
```

Comment:
We should have some convention around this. Begin/End blockers should probably return (preferably emit) tags and errors?

Reply:
I agree.
Let me put this into WIP and go through all methods.

Reply:
Ok, now all Begin/End Blocker should be consistent with returning errors